docs/mobility/slam/cartographer/_index.md

```diff
@@ -73,14 +75,25 @@ See Viam's [Pricing](https://www.viam.com/product/pricing) for more information.
 
   {{< /alert >}}
 
+#### Movement sensors (optional)
+
+You can use data from one or more movement sensors on your to supplement the required LiDAR data.
+If you choose to use movement sensor data for SLAM, you have three options:
```

Comment:
Nit: since all bullets begin with `Add`, we can make this whole section read as a sentence.

```suggestion
If you choose to use movement sensor data for SLAM, you can:
```

If instead you want to keep `you have three options:`, I'd recommend changing bullet points to:

> You can add only inertial ...
> You can add only odometry ...
> etc

Reply:
Okay I will apply but curious--is it best practice to make it read as a sentence? I sometimes prefer not to in part because it makes capitalization feel odd

Reply:
I'd be fine with either approach! I just feel that present `three options:` with items intro'd as `Add only` needs more fleshed-out form, like: `You can add only`. Starting options with `Add only` took me a few reads to understand who would be adding and where.

The "make it read as a sentence" approach I suggested was to avoid having to repeat `- You can` three times.

Definitely small, and you can certainly leave it as if if you like its current form! 👍